macro/src/lib.rs

```diff
@@ -3,8 +3,7 @@
 // see LICENSE for license details.
 
 //! Subxt macro for generating Substrate runtime interfaces.
-
-extern crate proc_macro;
+#![allow(clippy::manual_unwrap_or_default, missing_docs)]
```

Comment:
can you remove this `missing_docs` and add `missing_docs` on pub fn subxt?

```suggestion
// TODO: The workspace lint is not working properly so it's disabled here for now.
#![allow(clippy::manual_unwrap_or_default)]
```